packages/apps/datafrete/src/datafrete-webhook.ts

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable import/prefer-default-export */
+import type { Request, Response } from 'firebase-functions';
+import type { Orders } from '@cloudcommerce/types';
+import api from '@cloudcommerce/api';
+import config from '@cloudcommerce/firebase/lib/config';
+import functions from 'firebase-functions/v1';
+import logger from 'firebase-functions/logger';
+
+const handleWebhook = async (req: Request, res: Response) => {
+  // if (operatorToken !== req.get('x-operator-token')) {
+  //   return res.sendStatus(401);
+  // }
```

Comment:
Ué, vai quem quer? 😄

Reply:
Tirando isso aí você abriria pra qualquer pessoa alterar o status de qualquer pedido através desse endpoint, tem que manter a autenticação aqui e esse token pode vir de env (action input > env), se ele não for definido você usa a API key da Store API

Reply:
Adicionar um `datafrete-operator-token` ou coisa parecida aí https://github.com/ecomplus/cloud-commerce/blob/main/action.yml#L41 ...